Client/TeamTalkAndroid/src/main/java/dk/bearware/backend/TeamTalkService.java

```diff
@@ -407,9 +408,17 @@ private void displayNotification(boolean enabled) {
                     .setContentText(getNotificationText())
                     .build();
             }
-            notificationManager.notify(UI_WIDGET_TAG, UI_WIDGET_ID, widget);
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
@poretsky Why "Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q" for startForeground and "Build.VERSION.SDK_INT >= Build.VERSION_CODES.N" for stopForeground?